build/flaws/broken-links.js

```diff
@@ -43,7 +43,7 @@ function mutateLink(
   }
 ) {
   if (isSelfLink) {
-    $element.addClass("self-link");
+    $element.attr("aria-current", "true");
```

Comment:
@schalkneethling So if I understand you and Marco [in this comment](https://github.com/mdn/yari/pull/3866#discussion_r641422929) this here should be
```suggestion
    $element.attr("aria-current", "page");
```
...instead?

Reply:
Correct, thanks!